src/main/scala/models/Format.scala

```diff
@@ -0,0 +1,8 @@
+package CSVValidation
+
+case class Format(
+    maybeBooleanFormatOrRegExFormat: Option[String],
```

Comment:
shouldn't this be stuck into `pattern` since that's where it comes from?

Reply:
Done